Dockerfile.app

```diff
@@ -7,16 +7,16 @@ RUN curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
 
 WORKDIR /web
 
-# copy all dependency declerations inside the container
+# copy all dependency declarations inside the container
 COPY package*.json .
 COPY pnpm*.yaml .
```

Comment:
Don't try to be smart here, just list the files individually; its not like there is an enormous amount of them

Or copy all files (for locally building the containers I suggest adding a `.dockerignore` file to ignore `node_modules` etc.)